components/quality-dashboard/base/rbac/quality-dashboard.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: quality-dashboard-maintainer
+  namespace: quality-dashboard
+rules:
+  - verbs:
+      - create
+      - delete
+      - edit
+      - list
+    apiGroups:
+      - ''
+    resources:
+      - secrets
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: quality-dashboard-maintainers
+  namespace: quality-dashboard
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: konflux-qe-admins
```

Comment:
why this group need permissions to access all the secrets in the cluster?

Reply:
I agree, the cluster rule can be changed to a rule.

Reply:
thanks, addressed in https://github.com/redhat-appstudio/infra-deployments/pull/3370/commits/490a793af6ab59c2c2d77dafdcbc056381d24aa7